lib/vcr/configuration.rb

```diff
@@ -508,16 +507,31 @@ def load_library_hook(hook)
 
     def resume_fiber(fiber, response, hook_declaration)
       fiber.resume(response)
-    rescue FiberError
+    rescue FiberError => ex
       raise Errors::AroundHTTPRequestHookError.new \
-        "Your around_http_request hook declared at #{hook_declaration}" +
-        " must call #proceed on the yielded request but did not."
+        "Your around_http_request hook declared at #{hook_declaration}" \
+        " must call #proceed on the yielded request but did not. " \
+        "(actual error: #{ex.class}: #{ex.message})"
     end
 
-    def start_new_fiber_for(request, block)
-      Fiber.new(&block).tap do |fiber|
-        fiber.resume(Request::FiberAware.new(request))
+    def start_new_fiber_for(request, fibers, proc)
```

Comment:
Assignment Branch Condition size for start_new_fiber_for is too high. [17.52/15]